scipy/special/_add_newdocs.py

```diff
@@ -10778,7 +10778,7 @@ def add_newdoc(name, doc):
     Plot the function. For that purpose, we provide a NumPy array as argument.
 
     >>> import matplotlib.pyplot as plt
-    >>> x = np.linspace(-1, 1, 100)
+    >>> x = np.linspace(0, 1, 500)
```

Comment:
This is what we put in other places also maybe we don't need that many points?
```suggestion
    >>> x = np.linspace(0.01, 1, 200)
```